onnxruntime/core/optimizer/common_subexpression_elimination.cc

```diff
@@ -365,12 +407,19 @@ Status CommonSubexpressionElimination::ApplyImpl(Graph& graph, bool& modified, i
     for (const NodeArg* input_def : node->InputDefs()) {
       auto it = equivalence_classes.find(input_def);
       if (it == equivalence_classes.end()) {
+        const onnx::TensorProto* initializer_ = new onnx::TensorProto();
```

Comment:
>er_ [](start = 42, length = 3)

This is not a member

Reply:
Hi @yuslepukhin - thanks for the review! - I have made the fix for this